package-lock.json

```diff
@@ -1,8 +1,15108 @@
 {
   "name": "spotify",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
why has a new package.lock file been added?

Reply:
I searched in google about this issue and this is happening because of different versions of npms that we are using.


check this out. [StackOverflow](https://stackoverflow.com/questions/64813775/is-there-any-way-to-fix-package-lock-json-lockfileversion-so-npm-uses-a-specific)